.github/workflows/rasa-tests.yml

```diff
@@ -0,0 +1,103 @@
+name: Rasa CI
```

Comment:
Make sure that the other checks are also running.

Reply:
I am not sure what you mean with this, but I guess you resolved this right?